cyhy/core/geoloc.py

```diff
@@ -4,13 +4,15 @@
 
 import os
 import sys
+from netaddr import IPSet
 
 import geoip2.database
 from geoip2.errors import AddressNotFoundError
 
 GEODB_FILES = ["GeoIP2-City.mmdb", "GeoLite2-City.mmdb"]
 GEODB_CITY_PATHS = ["/usr/share/GeoIP/", "/usr/local/share/GeoIP/"]
 GEODB_FILE_PATHS = []
+RESTRICTED_COUNTRIES = ["China","Iran","North Korea","Russia"]
```

Comment:
Spaces are preferred between list items.  Normally we would run this whole file thru `black` to format it, but that is not necessary for this PR.
```suggestion
RESTRICTED_COUNTRIES = ["China", "Iran", "North Korea", "Russia"]
```